zenduty/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Since there are no service checks for this integration, the service_check file can be removed